### PR TITLE
Modernizing usage of Godeps

### PR DESCRIPTION
**Pull request by rafecolton**

### CODE REVIEW

**Comment by rafecolton** [comment]
@jszwedko I could use a sanity check on this one.  Attempting to use `bytes.Buffer` to get better error logging in cases where an `exec.Cmd` returns an error

File: clone_creator.go [hunk 1]
```diff
@@ -70,39 +80,57 @@ func (creator *clone) cloneRepo(dest string) error {
...
+				"git", "clone", "--depth", creator.Depth, repoURL.String(), dest,
+			},
+			Stderr: buff,
+		}
 	}
 
 	if err := cloneCmd.Run(); err != nil {
-		return err
-	}
+		logger.WithFields(logrus.Fields{
```
Comment (jszwedko) [inline clone_creator.go:100]: :+1:

File: clone_creator.go [hunk 2]
```diff
@@ -57,6 +60,13 @@ func (creator *clone) cloneNoCache() (string, error) {
 }
 
 func (creator *clone) cloneRepo(dest string) error {
+	git, err := fileutils.Which("git")
+	if err != nil {
+		return err
+	}
+
+	buff := &bytes.Buffer{}
```
Comment (jszwedko) [inline clone_creator.go:68]: Idiomatic Go might be to do `var buff bytes.Buffer`, but ¯_(ツ)_/¯

**Comment by jszwedko** [comment]
I approve of these changes

**Pull request merged**